hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMFileRequest.java

```diff
@@ -707,21 +707,21 @@ public static OmKeyInfo getOmKeyInfo(String volumeName, String bucketName,
       OmDirectoryInfo dirInfo, String keyName) {
 
     OmKeyInfo.Builder builder = new OmKeyInfo.Builder();
-    builder.setParentObjectID(dirInfo.getParentObjectID());
-    builder.setKeyName(keyName);
-    builder.setAcls(dirInfo.getAcls());
-    builder.addAllMetadata(dirInfo.getMetadata());
-    builder.setVolumeName(volumeName);
-    builder.setBucketName(bucketName);
-    builder.setCreationTime(dirInfo.getCreationTime());
-    builder.setModificationTime(dirInfo.getModificationTime());
-    builder.setObjectID(dirInfo.getObjectID());
-    builder.setUpdateID(dirInfo.getUpdateID());
-    builder.setFileName(dirInfo.getName());
-    builder.setReplicationConfig(RatisReplicationConfig
-        .getInstance(HddsProtos.ReplicationFactor.ONE));
-    builder.setOmKeyLocationInfos(Collections.singletonList(
-            new OmKeyLocationInfoGroup(0, new ArrayList<>())));
+    builder.setParentObjectID(dirInfo.getParentObjectID())
+            .setKeyName(keyName)
```

Comment:
These indents look like 8 spaces. They should be 4 to keep with the project style.

Reply:
Also I think you can simpliy this furher like:

```
return new OmKeyInfo.Builder()
    .setXyz()
    .setabc()
    .build();
```

Reply:
Fixed.